src/controllers/notice.expired.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Session } from "ch-node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+import {
+  retrieveCompanyProfileFromObjectionSession,
+} from "../services/objection.session.service";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  const session: Session = req.session as Session;
+
+  const { companyName } = retrieveCompanyProfileFromObjectionSession(session)
+
+  return res.render(Templates.NOTICE_EXPIRED, {
+    companyName,
+    templateName: Templates.NOTICE_EXPIRED,
+  });
```

Comment:
I think this controller will need it's own unit test, so we can mock retrieveCompanyProfileFromObjectionSession. to return a number and we can test that the number appears on the page.